style.css

```diff
@@ -114,19 +130,204 @@ body {
-  padding: 0;
-  margin: 0;
+.dark-bg ul {
   align-self: flex-start;
 }
 
-.headline ul li {
+.dark-bg ul li {
   display: inline-block;
 }
 
-.headline .squares {
+.dark-bg .squares {
   position: absolute;
   right: 0;
   bottom: 0;
 }
+
+/*  ---- Recent work ---- */
+
+.recent-work {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  align-items: center;
+}
+
+.title {
+  width: 85%;
+  padding-top: 130px;
+  margin-bottom: 100px;
+  text-align: center;
+}
+
+.title h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 50px;
+}
+
+.title hr {
+  width: 100%;
+}
+
+.projects {
+  justify-items: center;
+  background: #fff;
+}
+
+.project {
+  width: 85%;
+  height: 386px;
+  position: relative;
+  margin-bottom: 30px;
+}
+
+.project a {
+  color: #fff;
+  height: 48px;
+  text-decoration: none;
+  background: #ff6b00;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+}
+
+.project-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 568px;
+  color: #091e42;
+}
+
+.project-1 .project-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.project-1 img {
+  width: 100%;
+}
+
+.project-1 ul {
+  padding-bottom: 50px;
+}
+
+.project-1 ul li {
+  background: #fff;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+}
+
+.project-2 ul li {
+  background: rgba(255, 255, 255, 0.24);
+  padding: 10px 12px;
+}
+
+.project-1 .project-skills li {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.project ul {
+  display: flex;
+  width: 100%;
+  gap: 12px;
+}
+
+.project-2 {
+  color: #fff;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(Media/background-project.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 0 20px 90px 20px;
+  display: flex;
+  justify-content: end;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.project-2 .project-title {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+/* ---- about me ---- */
+
+.about-me {
+  gap: 160px;
+}
+
+.about-me .about-me-img {
+  position: absolute;
+  right: 0;
+  top: 160px;
+}
+
+.get-my-resume {
+  display: flex;
+  text-align: center;
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  text-decoration: none;
+  align-items: center;
+}
+
+.get-my-resume p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.content-2 {
+  height: 348px;
+}
+
+.about-me hr {
+  width: 80%;
+}
+
+.skills {
+  display: flex;
+  flex-direction: column;
```

Comment:
There is a project-skills class already so doing another style called skills might be confusing to read, I think it would be better if you rename one of those classes so it's clear which part of the code is referring to